core/src/tangram.h

```diff
@@ -88,12 +88,11 @@ float getTilt();
 
 // Transform coordinates in screen space (x right, y down) into their longitude
 // and latitude in the map view
```

Comment:
Can you update the comments here to reflect (`lng, lat`) pair has the appropriate value. Currently if one just reads the comments it seems `x` and `y` will be transformed/updated to have `lng`/`lat` respectively.